### PR TITLE
Template validation error message

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
When server side validation fails we need to report on this in the correct manner

# What has changed
Reports back print template validation fails to the end user

# How to test?
Fire it up,  poke some bad validation at it like existing packcodes or templates like these [""].  I had to temp alter the java code to make an error for unknown supplier as the frontend limits this.

# Links
https://trello.com/c/2tuhw4KJ/2747-response-ops-create-template-validation-error-messages-are-not-being-passed-through-to-the-ui-5

# Screenshots (if appropriate):

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "configurations": [
+        {
+            "type": "pwa-chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
+            "url": "http://localhost:3000",
+            "webRoot": "${workspaceFolder}"
+        }
+    ]
+}
```
Comment (ryangrundy7) [inline .vscode/launch.json:15]: Don't think you meant to push this?

File: build.sh [hunk 2]
```diff
@@ -9,4 +9,6 @@ cd ..
 cp -r ui/build/* src/main/resources/static
 rm -r ui/build/* || true
 
-mvn clean install
\ No newline at end of file
+# Put back to just: mvn clean install
+#mvn clean install
+mvn clean install -Dmaven.test.skip=true
```
Comment (ryangrundy7) [inline build.sh:14]: Needs to be back to mvn clean install

File: .vscode/launch.json (hunk 1 above)
Comment (LukeLoze1) [reply .vscode/launch.json:15]: ta, will add to ignore

**Review by NickGrantONS — CHANGES_REQUESTED** [review]
Use `Optional` instead of returning `null` or a value

**Review by ryangrundy7 — APPROVED** [review]
Looks good 👍

**Review by hbrace — APPROVED** [review]

**Review by NickGrantONS — APPROVED** [review]

**Pull request merged**